src/deepsparse/transformers/pipelines/pipeline.py

```diff
@@ -135,7 +135,7 @@ def tokens_to_engine_input(
         if not all(name in tokens for name in self.onnx_input_names):
             raise ValueError(
                 f"pipeline expected arrays with names {self.onnx_input_names}, "
-                f"received inputs: {list(tokens.keys())}"
+                f"received inputs: {list(tokens[0].keys())}"
```

Comment:
bumping this - the if check expects tokens to be a dict here, but we seem to be expecting a list in the error message. should we actually be unwrapping a potential list before the if statement?

Reply:
Actually my new code no longer calls into this func, so just reverted it